manifests/config/audit_profiles.pp

```diff
@@ -9,25 +9,31 @@
 # - `00_head.rules`:  Contains `auditctl` general configuration to
 #   remove existing rules when the rules are reloaded, ignore rule
 #   load errors/failures, and set the buffer size, failure mode,
-#   and rate limiting.
+#   and rate limiting
 # - `05_default_drop.rules`: Contains filtering rules for efficiency
 #   - Rules to drop prolific events of low-utility
 #   - Rules to restrict events based on `auid` constraints that would
 #     normally be applied to all rules
+# - `40_custom.rules`: Custom rules as defined by the ``auditd::custom_rules``
```

Comment:
I did not see `auditd::custom_rules` anywhere.  Looks like cruft from an earlier implementation?